client/python/cryoet_data_portal/src/cryoet_data_portal/templates/Model.jinja2

```diff
@@ -47,8 +50,9 @@ class {{ model.name }}(Model):
         """
         Examples:
             Get an {{ model.name }} by ID:
+
             >>> {{ model.model_name_underscores }} = {{ model.name }}.get_by_id(client, 1)
-                print({{ model.model_name_underscores }}.name)
+            >>> print({{ model.model_name_underscores }}.name)
```

Comment:
This issue predates the changes in the PR, all models don't have the name field, example: TiltSeries, PerSectionAlignmentParameters, etc. 

A simple workaround for this would be to print id in all cases. Or only print name in the cases where the name is a field of the model.

Reply:
Let me know if this is what you mean!